### PR TITLE
Feature/select picture microblog

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: commons-component-common/src/main/java/org/exoplatform/commons/utils/HTMLSanitizer.java [hunk 1]
```diff
@@ -134,6 +142,11 @@
                                                                                                                                 .allowAttributes("src")
                                                                                                                                 .matching(ONSITE_OR_OFFSITE_URL)
                                                                                                                                 .onElements("img")
+                                                                                                                                .allowAttributes("src")
+                                                                                                                                .matching(ONSITE_OR_OFFSITE_URL)
+                                                                                                                                .onElements("img")
+                                                                                                                                .allowAttributes("referrerpolicy")
```
Comment (thomasdelhomenie) [inline commons-component-common/src/main/java/org/exoplatform/commons/utils/HTMLSanitizer.java]: Just curious, where is this attribute used ? By the ckeditor plugin ?

File: commons-extension-webapp/src/main/webapp/ckeditor/plugins/filetools/dev/uploaddebugger.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or http://ckeditor.com/license
+ */
+
+'use strict';
+
+// Slow down the upload process.
+// This trick works only on Chrome.
```
Comment (thomasdelhomenie) [inline commons-extension-webapp/src/main/webapp/ckeditor/plugins/filetools/dev/uploaddebugger.js]: Do we need this file for the feature ? Or is it only for debugging purpose ?

File: commons-extension-webapp/src/main/webapp/ckeditor/plugins/imageresize/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+ImageResize Plugin for CKEditor 4
```
Comment (thomasdelhomenie) [inline commons-extension-webapp/src/main/webapp/ckeditor/plugins/imageresize/README.md]: README should be removed IMO.

File: commons-extension-webapp/src/main/webapp/javascript/eXo/commons/UISelectFromDrives.js [hunk 4]
```diff
@@ -0,0 +1,371 @@
+(function($, base){
```
Comment (thomasdelhomenie) [inline commons-extension-webapp/src/main/webapp/javascript/eXo/commons/UISelectFromDrives.js]: We now have a pure js version of the "select from drive" component, right ? If so, we should open an issue (improvement) to use it for File activities.

File: commons-component-common/src/main/java/org/exoplatform/commons/utils/HTMLSanitizer.java (hunk 1 above)
Comment (boubaker) [reply commons-component-common/src/main/java/org/exoplatform/commons/utils/HTMLSanitizer.java]: Yep, it's used when using external links in img to remove Remove referer  http Header when requesting the image, else the external sites will reject the HTTP request.

File: commons-extension-webapp/src/main/webapp/ckeditor/plugins/filetools/dev/uploaddebugger.js (hunk 2 above)
Comment (boubaker) [reply commons-extension-webapp/src/main/webapp/ckeditor/plugins/filetools/dev/uploaddebugger.js]: This was delivered by in the plugin bundle, as you can see it's added as it is in predefined CKEditor plugins.

File: commons-extension-webapp/src/main/webapp/ckeditor/plugins/imageresize/README.md (hunk 3 above)
Comment (boubaker) [reply commons-extension-webapp/src/main/webapp/ckeditor/plugins/imageresize/README.md]: This was delivered by in the plugin bundle, as you can see it's added as it is in predefined CKEditor plugins. Why removing it if it's delivered like that as external third party library ?

File: commons-extension-webapp/src/main/webapp/javascript/eXo/commons/UISelectFromDrives.js (hunk 4 above)
Comment (boubaker) [reply commons-extension-webapp/src/main/webapp/javascript/eXo/commons/UISelectFromDrives.js]: Yep, that's the purpose. We may have to add different parameters, but it'a the idea.

File: commons-extension-webapp/src/main/webapp/ckeditor/plugins/filetools/dev/uploaddebugger.js (hunk 2 above)
Comment (thomasdelhomenie) [reply commons-extension-webapp/src/main/webapp/ckeditor/plugins/filetools/dev/uploaddebugger.js]: ok

File: commons-extension-webapp/src/main/webapp/javascript/eXo/commons/UISelectFromDrives.js (hunk 4 above)
Comment (thomasdelhomenie) [reply commons-extension-webapp/src/main/webapp/javascript/eXo/commons/UISelectFromDrives.js]: ok :+1: I let you open this issue.

Comment (boubaker) [reply commons-extension-webapp/src/main/webapp/javascript/eXo/commons/UISelectFromDrives.js]: done PLF-7680

**Pull request merged**